src/frontend/components/_features/[workspace]/editor/diff-viewer/file-diff-view.tsx

```
@@ -6,8 +6,8 @@
  */
 
 import { DiffEditor } from '@monaco-editor/react'
-
 import { GraphicalDiffViewer, isGraphicalFile } from './graphical-diff-viewer'
+import { useDiffEditorTeardown, useDiffModelPaths } from './use-diff-editor-teardown'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)